NaviGator/hardware_drivers/navigator_drone_comm/package.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>navigator_drone_comm</name>
+  <version>0.0.0</version>
+  <description>The navigator_drone_comm package</description>
+
+  <maintainer email="exla@todo.todo">exla</maintainer>
```

Comment:
exla is cute but this might be better for whoever comes upon this in the future

```xml
<maintainer email="alexjohnson@todo.todo">Alex Johnson</maintainer>
```